mmv1/third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -3306,7 +3306,7 @@ func expandAutoProvisioningDefaults(configured interface{}, d *schema.ResourceDa
 
 func expandAuthenticatorGroupsConfig(configured interface{}) *container.AuthenticatorGroupsConfig {
 	l := configured.([]interface{})
-	if len(l) == 0 {
+	if len(l) == 0 && l[0] != nil {
```

Comment:
I'm a little confused by this - if `len(l) == 0` wouldn't `l[0]` not exist because `l` is empty? Should this be `||`?